extensions/cdc-mysql/src/test/java/com/hazelcast/jet/cdc/mysql/AuthAndConnectionIntegrationTest.java

```diff
@@ -0,0 +1,143 @@
+/*
+ * Copyright (c) 2008-2020, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hazelcast.jet.cdc.mysql;
+
+import com.hazelcast.jet.JetInstance;
+import com.hazelcast.jet.Job;
+import com.hazelcast.jet.cdc.ChangeRecord;
+import com.hazelcast.jet.core.JetTestSupport;
+import com.hazelcast.jet.pipeline.Pipeline;
+import com.hazelcast.jet.pipeline.Sinks;
+import com.hazelcast.jet.pipeline.StreamSource;
+import com.hazelcast.test.annotation.NightlyTest;
+import java.sql.SQLException;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+import org.testcontainers.containers.MySQLContainer;
+
+import static com.hazelcast.jet.core.JobStatus.FAILED;
+import static com.hazelcast.test.HazelcastTestSupport.assertEqualsEventually;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+import static org.testcontainers.containers.MySQLContainer.MYSQL_PORT;
+
+@Category(NightlyTest.class)
+public class AuthAndConnectionIntegrationTest extends JetTestSupport {
```

Comment:
The tests you describe here are conceptually non-DB specific, but I'm not sure you can rely on every connector to every DB behaving the same way, so it would be safest to threat it as MySQL specific tests.

Reply:
Threat it and NAME it like MySQL specific tests.

Reply:
This test is in cdc-mysql module. Do we need to mention MySQL also in test name?